corehq/apps/users/management/commands/create_billing_admin_role.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+from corehq.apps.domain.models import Domain
+from corehq.apps.users.models import UserRole, UserRolePresets
+
+
+class Command(BaseCommand):
```

Comment:
in general I prefer to add migrations instead of management commands, to ensure that it gets run on every environment. removes the possibility of user error and lets a dev confirm that it has been run via the migration history.